### PR TITLE
First branch

### PR DESCRIPTION
**Pull request by vamsikrishna0208**
the static web site has been create

### CODE REVIEW

**Comment by vamsikrishna0208** [comment]
the code has been sent for review

File: webpage.css [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    padding: 12px 30px;
+    color: #fff;
+    font-size: 14px;
+    font-weight: 300;
+    text-decoration: none;
+    border-radius: 33px;
+    background-color: #00bcd4 ;
+    font-family: Arial, Helvetica, sans-serif;
+}
+.about-text h3 {
```
Comment (vamsi15) [inline webpage.css]: add h3 font-size to 40px

File: webpage.css [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+}
+.btn-learn:hover{
+    color: #ffffff;
+}
+.about-image img{
+    width: 95%;
+    min-width: 95%;
+}
+.about-text {
+    margin: 100px 0px;
```
Comment (vamsi15) [inline webpage.css]: change top-bottom margin to 95px

File: webpage.css (hunk 1 above)
Comment (vamsikrishna0208) [reply webpage.css]: font-size changed to 40px

File: webpage.css [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+.beliefs img{
+    width: 95%;
+    min-width: 95%;
+}
+.domain-slider img{
+    width: 95%;
+    min-width: 95%;
+}
+#beliefs {
+    margin: 100px 0px;
```
Comment (vamsi15) [inline webpage.css]: change top-bottom margin to 95 px

Comment (vamsikrishna0208) [reply webpage.css]: margin changed to 95px

File: webpage.css (hunk 2 above)
Comment (vamsikrishna0208) [reply webpage.css]: changed top-bottom margin to 95px

**Pull request merged**